### ISSUE TITLE
[Accessibility][Object Information - Start Bot]: 'Reloading' Bot information is not provided by screen reader.

### ISSUE THREAD

**Issue by ShellySoni** [microsoft/BotFramework-Composer#2068]
**User Experience:**
Screen reader users will not get to know the present status if while reloading the bot, its information is not get conveyed to user.

**Test Environment:**
Environment: Chromium Edge (Anaheim) + Narrator
Build Version: 2004 (OS Build 19564.1005)​​
Microsoft Edge Version: 82.0.425.0 (Official build) canary (64-bit)​
Test URL: http://localhost:3000/home

**Repro Steps:**
1. Open URL: http://localhost:3000/home
2. "Bot Framework Composer" homepage will gets opened.
3. Select any of the previously created bot from "Recent Bots" or create one from the available Bots under 'Examples' and activate it.
4. Navigate to the "Start" button available at the header and activate it.
5. Observe the issue.

**Actual Result:**
 On clicking on 'Start' button, 'Reloading' Bot information is not provided by screen reader.

**Expected Result:**
On clicking on 'Start' button, 'Reloading' Bot information should be provided by screen reader.

**MAS Reference:**
https://microsoft.sharepoint.com/:w:/r/sites/accessibility/CELA/Official_Microsoft_Accessibility_Standards/MAS%204.2.1%20%E2%80%93%20Object%20Information.docx?d=weee97bbd999048a08b036ea775581aa8&csf=1&e=UYEfO3

**Attachment:**
[MAS4.2.1_Start Bot_Reloading.zip](https://github.com/microsoft/BotFramework-Composer/files/4244375/MAS4.2.1_Start.Bot_Reloading.zip)

**Comment by ashish315** [comment]
@corinagum As checked, the narrator still not announces the information about the 'Reloading'. 
We checked on the below environment and the latest 'Master' build downloaded on 23rd April.
**Test Environment**
Browser: Microsoft Edge Dev {Version 84.0.488.1 (Official build) dev (64-bit)}
OS build: 2004 (19562.1000)
Screen Reader: Narrator
URL: [Bot Framework Composer](http://localhost:3000/dialogs/Main)
[**Prerequisite Here**](https://github.com/microsoft/BotFramework-Composer)
![Github](https://user-images.githubusercontent.com/60913131/80138432-6f8eca80-85c2-11ea-88d4-e617ae22ae4f.png)

**Comment by ashish315** [comment]
As checked the issue is resolved. So, HCL is closing the bug.
